### DISCUSSION TITLE
Question about MallocAllocator::allocateContiguousImpl

### DISCUSSION THREAD

**Discussion by waitinfuture** [facebookincubator/velox#3865] [Q&A] [answered]
I don't understand in MallocAllocator::allocateContiguousImpl why numMapped_ adds ```numNeededPages + numContiguousCollateralPages``` instead of numPages, since the following mmap actually maps numPages pages.

cc @xiaoxmeng

**Comment by xiaoxmeng** [comment, marked as answer]
@waitinfuture I think you are right. We shall add numPages. Thanks for pointing out!

**Comment by mbasmanova** [comment]
@xiaoxmeng Meng, is this already fixed. Would you add a reference to the PR?
